src/libmongoc/tests/unified/operation.c

```diff
@@ -851,14 +852,22 @@ operation_find (test_t *test,
       goto done;
    }
 
+   opts = bson_copy (bson_parser_get_extra (parser));
+   if (op->session) {
+      if (!mongoc_client_session_append (op->session, opts, error)) {
```

Comment:
I was surprised to discover that the "session" argument was not being handled for all operations. See: [CDRIVER-3964](https://jira.mongodb.org/browse/CDRIVER-3964).

Reply:
Good catch, and thank you for filing a ticket to track this.